proposals/0002-root-signature-in-clang.md

```diff
@@ -504,20 +510,20 @@ The additional semantic rules not already covered by the grammar are listed here
   - DESCRIPTORS_VOLATILE
```

Comment:
Need to mention version requirements for DESCRIPTOR_RANGE_FLAGS on samplers too.